modules/tinymce/src/core/main/ts/util/Quirks.ts

```diff
@@ -438,6 +438,16 @@ const Quirks = (editor: Editor): Quirks => {
     });
   };
 
+  /**
+   * This is a helper function to test if the selection is all the content available
+   */
+  const isAllContentSelected = (editor: Editor): boolean => {
+    // The next three lines address TINY-4550
```

Comment:
Maybe move this to the comment below instead at the call site and maybe explain why it's there that we check if all the contents is selected to avoid it from normalizing inwards to an image not sure about the wording.